### PR TITLE
Fix/travis

### PR DESCRIPTION
**Pull request by Br3nda**
Fixes all except for these two:
```
Failures:
  1) RebateForm signed in as dia #show can see edit link
     Failure/Error: expect(page).to have_text('Income declaration (before tax)')
       expected to find text "Income declaration (before tax)" in "Te Whakamāmā i ngā Reiti\nRates Rebates\nanalytics rebate forms batches councils users ernestoschultz@leannon.info Sign Out\nCustomer Details\nSignature required"
     Shared Example Group: "can edit" called from ./spec/features/admin/rebate_forms/edit_spec.rb:45
     # ./spec/features/admin/rebate_forms/edit_spec.rb:35:in `block (4 levels) in <top (required)>'
     # ./spec/rails_helper.rb:50:in `block (3 levels) in <top (required)>'
     # ./spec/rails_helper.rb:49:in `block (2 levels) in <top (required)>'
  2) RebateForm signed in as council #show can see edit link
     Failure/Error: expect(page).to have_text('Income declaration (before tax)')
       expected to find text "Income declaration (before tax)" in "Te Whakamāmā i ngā Reiti\nRates Rebates\nanalytics rebate forms batches councils users jackson@dach.name Sign Out\nCustomer Details\nSignature required"
     Shared Example Group: "can edit" called from ./spec/features/admin/rebate_forms/edit_spec.rb:53
     # ./spec/features/admin/rebate_forms/edit_spec.rb:35:in `block (4 levels) in <top (required)>'
     # ./spec/rails_helper.rb:50:in `block (3 levels) in <top (required)>'
     # ./spec/rails_helper.rb:49:in `block (2 levels) in <top (required)>'
Finished in 29.87 seconds (files took 4.24 seconds to load)
277 examples, 2 failures, 22 pending
Failed examples:
rspec ./spec/features/admin/rebate_forms/edit_spec.rb[1:2:2:1] # RebateForm signed in as dia #show can see edit link
rspec ./spec/features/admin/rebate_forms/edit_spec.rb[1:3:2:1] # RebateForm signed in as council #show can see edit link
```

### CODE REVIEW

**Comment by mischa-s** [comment]
amazing! thanks! I'll fix those, they are on my list

**Comment by Br3nda** [comment]
I've moved the env variables to the travis.yaml. This means you can turn on travis for your fork and everything will work without more setup needed.

**Comment by Br3nda** [comment]
@mischa-s note this PR doesn't have the latest from your other PR #271

File: .travis.yml [hunk 1]
```diff
@@ -2,37 +2,27 @@ sudo: required
...
-  - bundle exec rake db:migrate
-  - bundle exec rake assets:precompile
-  - restart
 env:
   global:
-    secure: T7+npfKb6EW0G++OP4C91y2WMrMYyOrozeNT1WPCuedd5vKdILSskMz5LBav54F0J6OBbwozH82e0xclMTa23VuOrc9805tEcwtSRdDYb/mIb7F6gEyseqLIk/TZvDfmB5Yrrji0IVX9s5q0OKfVVb0rTrgh87pSRW2YnYAu41k6v2pK/OquixQX0sogblOLfKYprqh4h3Ay6+1IIDmC/lg0XSe+0xRFBZ5FJk0ycStu74xM+qbphIFgzd6q2kChFqRrOporeCgN3WSAFsQd7QneLITN3eucH9AR0OC9Udpuxr8tvAqTfy7Lo3ogj9PTYjD60ObQjcr212d/isSaZd6Zk2wHH35bzsZU3Y5f6vsILiZVJY2rDPkf7A6jMrWSPw8jd/CRizFjedveF9jCy3J2viwNI23y4Ce+sQkc5u0dAV3Z7n5Avy1R/UjyBpRoYb7VhfX1mZTKsVfLPoOxMuvChyOAL4MZGR83N9emHoLFL1r/JsRlqtun/F8i7vsu3OmqpO9ZcexEh+jx1Hqt6qSizNto6bQkKQ5WOul0kfT9U4kIHcKe0EIsUmD2d6CYLXBIYuSitp3/Fq/yX/tUwm9tJED+J0g76g8ItFAznf+7nYe4MSh039ehOKXLW4v+w6624RYfmjCauNHb4HSmwE0n2y8XgLwat7Tj1hwV8Qc=
+  - PGPORT=5433
+  - YEAR=2018
+  - SIGNING_URL="http://127.0.0.1:4000/#/{token}"
+  - OPENFISCA_ORIGN="https://openfisca.ratesrebates.services.govt.nz/calculate"
```
Comment (Br3nda) [inline .travis.yml]:
```suggestion
  - OPENFISCA_ORIGN="https://govtnz-rates-rebate-of.herokuapp.com/" 

```

**Pull request merged**